### ISSUE TITLE
Documentation not specific about multiple register

### ISSUE THREAD

**Issue by benzen** [hotwired/stimulus#334]
I understand that basecamp use webpack to build assets like stimulus controllers.
By doing so i assume that the js payload is kept at it's minimum.

But I've made diffrent trade-off and it's unclear to me if i'm doing something wrong. I think that the doc could be a bit more specific to help get a better understanding of what is happing.

I use turbolinks in conjunction stimulus.

In each page that require a stimulus controller here is what I'm doing at the moment 

```js
$(document).on('turbolinks:load', () => {
      APP.register('something', class extends Stimulus.Controller {...})
})
    
```

So basically each time a user enter this page, the controller is registered once more. Is this dommageable in any way?
Do I end up with two controller working on the same page/ same elements ?

Should I unregister my controller ? Is that what Application#unload is for ?

**Comment by dhh** [comment]
You might want to have a look at the stimulus-rails autoloader. Sounds similar to what you're trying to do: https://github.com/hotwired/stimulus-rails/blob/main/app/assets/javascripts/stimulus/loaders/autoloader.js
